### DISCUSSION TITLE
[2483830]crazy sprint[仅本体]

### DISCUSSION THREAD

**Discussion by jack20211** [pjy612/SteamManifestCache#278] [Share]
`appid:2483830`

Key:
``` 
2483831;6fdbe49aba445ea4a1c6593768e77a8050621304211c43d2f3e16b47feb2ecf3
``` 

[2483830_config.vdf.txt.txt](https://github.com/pjy612/SteamManifestCache/files/13837033/2483830_config.vdf.txt.txt)

[crazy sprint.zip.txt](https://github.com/pjy612/SteamManifestCache/files/13837323/crazy.sprint.zip.txt)

code:E31816B2F0F7EEC3

email:[doufeng2006@gmail.com](mailto:doufeng2006@gmail.com)

**Comment by pjy612** [comment]
未收录 麻烦附带清单

**Reply by jack20211** [reply]
好了， 已经传了清单，看一下对不对

**Reply by jack20211** [reply]
收到，谢谢大佬
